aws/s3.tf

```diff
@@ -123,3 +123,94 @@ module "s3_bucket" {
 
   tags = local.tags
 }
+# aws S3 backup //////////////////////
+resource "aws_backup_vault" "datagrok_public_vault" {
+  name = var.s3_backup_vault_name
+}
+
+resource "aws_iam_role" "backup_role" {
+  name                = "s3-backup-role"
```

Comment:
The name should be with the same prefixes as other resources to determine that it was created for our resources
Like for the logging bucket https://github.com/datagrok-ai/tf-module-datagrok-core/pull/26/files#diff-bfa811139a4cedd8c743561172e24c14f800b2fd054b137f984598b3d4195706R10